roles/docker-rootless/tasks/main.yml

```diff
@@ -61,8 +61,7 @@
 
 - name: Execute setup rootless docker script
   become: true
-  shell:
-    /tmp/rootless_docker_setup.sh
+  command: /tmp/rootless_docker_setup.sh
```

Comment:
Has this been tested? Sometimes shell is needed for parsing output, although that does not appear to be the case here.